packages/hash/external-services/docker-compose.yml

```diff
@@ -32,7 +36,7 @@ services:
     #   - ../../../var/external-services/opensearch/data:/usr/share/opensearch/data
 
   postgres:
```

Comment:
I think we also want `init: true` on postgress. I think not everyone uses `detach`ed mode for the compose, so if having init makes ctrl+c possible with alpine, we should add that

Reply:
Somehow, for `postgres` this is not needed. It will quit without issues (Don't ask why 😄).

Reply:
Alright, perhaps they manually trap the signal :shrug: